core/sis-utility/src/main/java/org/apache/sis/measure/Units.java

```diff
@@ -1290,6 +1303,13 @@
         DECIBEL = add(bel, Prefixes.converter('d'), "dB", ACCEPTED, (short) 0);
         UNITY   = UnitRegistry.init(one);  // Must be last in order to take precedence over all other units associated to UnitDimension.NONE.
 
+        /*
+         * Acceleration units
+         */
+        METRES_PER_SECOND_SQUARED = mps2;
+        mps2.related(1);
+        GAL = add(mps2, centi, "gal", (byte) (OTHER | PREFIXABLE | ACCEPTED), (short) 0);
```

Comment:
We have a tricky situation here. According Wikipedia (I verified also in BIPM), the symbol is "Gal" (upper-case letter) but the name is "gal" (lower-case letter). The argument in `add` method call is the symbol, so it should have an upper-case letter. But we need to add an entry in the localized resource files for formatting the name with lower-case letter when writing as text.

Another topic is that this units is part of the centimeter–gram–second (CGS) system. This is the first occurrence of this system in our `Units` class, but CGS may be important enough to deserve a new constant in `UnitRegistry` that we should use instead of `OTHER`.